src/Http/Http.Abstractions/src/RequestDelegateResult.cs

```diff
@@ -23,7 +23,8 @@ public RequestDelegateResult(RequestDelegate requestDelegate, IReadOnlyList<obje
     public RequestDelegate RequestDelegate { get; }
 
     /// <summary>
-    /// Gets endpoint metadata inferred from creating the <see cref="RequestDelegate" />
+    /// Gets endpoint metadata inferred from creating the <see cref="RequestDelegate" />. This does not contain
```

Comment:
This comment is a little confusing only because `InitialEndpointMetadat` no longer exists, correct?